### DISCUSSION TITLE
Mirth Client API - Delete Channel Group

### DISCUSSION THREAD

**Discussion by boukenka** [nextgenhealthcare/connect#5105] [Q&A] [answered]
Hello,

I don't see anywhere in NextGen Connect Client API swagger 3.12.0 the ability/possibility to delete a channel group.
It is only possible to delete a channel.
Am I wrong? Is there a possibility to do this? Or will this be implemented in the future?

**Comment by jonbartels** [comment, marked as answer]
The Channel Group API only supports updating ALL the channel groups in a batch with `/channelgroups/_bulkUpdate` . So to delete a group you have to get the whole list then remove the one you want to remove then POST the new complete list.

The last comment in this old issue from narupley describes whats going on: https://github.com/nextgenhealthcare/connect/issues/4084
